### PR TITLE
Add sveltepress

### PR DESCRIPTION
**Pull request by Blackman99**

### CODE REVIEW

File: src/routes/templates/templates.json [hunk 1]
```diff
@@ -897,5 +897,15 @@
 		"category": "Svelte",
 		"tags": ["storybook", "typescript"],
 		"stars": 0
+	},
+	{
+		"title": "Sveltepress",
+		"url": "https://github.com/SveltePress/sveltepress",
+		"description": "A markdown centered site build tool with full power of sveltekit.",
+		"npm": "@sveltepress",
```
Comment (MacFJA) [inline src/routes/templates/templates.json]: This is not a valid NPM package name

**Review by MacFJA — CHANGES_REQUESTED** [review]

**Comment by Blackman99** [comment]
The valid npm command is `npm create @sveltepress` not `npm install @sveltepress`
However there's no such an option

File: src/routes/templates/templates.json (hunk 1 above)
Comment (Blackman99) [reply src/routes/templates/templates.json]: The valid npm command is `npm create @sveltepress` not `npm install @sveltepress`
However there's no such an option

Comment (MacFJA) [reply src/routes/templates/templates.json]: Maybe we should change the NPM/Yarn/PNPM command to `create` on the templates pages ? :thinking: 

I have to check if this change doesn't create a side effect for the existing templates

Comment (Blackman99) [reply src/routes/templates/templates.json]: That would be great!

**Review by lachlancollins — APPROVED** [review]

**Pull request merged**